docs/assets/joining_parallel_gateway_doc.bpmn

```diff
@@ -0,0 +1,233 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_1a8cbo5" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.0.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.0.0">
+  <bpmn:process id="Process_0zaz4fy" isExecutable="true">
+    <bpmn:task id="Activity_0np5087" name="Increment number of taken sequence flows">
```

Comment:
All the other tasks are the names of records (I assume these were meant as this record is written or processed or so), but this really describes a task as a verb or action. Even though this is preferred for the naming of activities, it seems out of place in the current documentation.

🔧 Please reconsider how we can make it more clear what the difference is between writing records (as the result of processing a command) and making state changes (as the result of applying an event) in this model. In the end, our engine works through these 2 mechanisms, and it would be great if we could describe the logic in those terms.